src/main/java/List.java

```diff
@@ -0,0 +1,61 @@
+public class List {
+    private static int amountOfItems = 0;
+    private static Task[] tasks;
+
+
+    public List() {
+        tasks = new Task[100];
+    }
+
+    public int getListSize() {
+        return amountOfItems;
+    }
+
+    public void addTask(String input) {
+        if (input.contains("/by") || input.split(" ")[0].equals("deadline")) {
+            if (input.split(" ", 2)[0].equals("deadline")) {
+                input = input.split(" ", 2)[1];
```

Comment:
Avoid recycling variable. Consider declaring another variable `String description` to store instead of using `input` again.